explorer/explorer_pods.go

```diff
@@ -40,6 +69,22 @@ func (n *PodsExplorer) List() error {
 	return nil
 }
 
+// gets pods from map of owners
+func getOwnerPods(pods []apiv1.Pod, owners map[string]bool, resourceKind string) []apiv1.Pod {
```

Comment:
So getOwnerPods looks pretty solid

My one nitpick is that the first argument looks to be simplifying what we pass to the function but is actually introducing more complexity. Would it be saner just to pass `pods` and not `pods.Items` to the function so we have more standardized input?

Reply:
Also `resourceKind` could probably have a different name

maybe `resourceKindToMatch` or something